app/models/billboard.rb

```diff
@@ -30,6 +30,8 @@ class Billboard < ApplicationRecord
   attribute :target_geolocations, :geolocation_array
   enum display_to: { all: 0, logged_in: 1, logged_out: 2 }, _prefix: true
   enum type_of: { in_house: 0, community: 1, external: 2 }
+  enum render_mode: { forem_markdown: 0, raw: 1 }
+  enum template: { authorship_box: 0, plain: 1 }
```

Comment:
can you clarify what is meant here by template? and what is `authorship_box`?

Reply:
@maestromac `Template` is the wrapping HTML, so "authorship box" is the current template for billboards where we show the name/image of the org and/or the site name, etc.

"plain" is no visible template wrappe, but it still contains the div which stores the info to track events.

I used `template` to match the similar functionality we have with `pages` where we can either wrap the content in a certain context or just have it cover the whole page.